template.distillery/demo_spinner.eliom

```diff
@@ -0,0 +1,47 @@
+(* This file was generated by Ocsigen Start.
+   Feel free to use it, modify it, and redistribute it as you wish. *)
+
+(* Spinner demo *)
+
+(* Service for this demo *)
+let%server service =
+  Eliom_service.create
+    ~path:(Eliom_service.Path ["demo-spinner"])
+    ~meth:(Eliom_service.Get Eliom_parameter.unit)
+    ()
+
+(* Make service available on the client *)
+let%client service = ~%service
+
+(* Name for demo menu *)
+let%shared name = "Spinner"
+
+(* Class for the page containing this demo (for internal use) *)
+let%shared page_class = "os-page-demo-spinner"
+
+(* Build the spinner *)
+let%client make_spinner () =
+  (* [Ot_spinner.with_spinner_no_lwt] accepts an Lwt thread "slowly"
+     producing HTML content *)
+  Ot_spinner.with_spinner_no_lwt
+    (* sleep for 5 seconds to simulate a delay, then return content *)
+    (let%lwt () = Lwt_js.sleep 500. in
```

Comment:
Right, thanks :). I used a long delay for debugging and forgot to change it back.